config.dev.toml

```diff
@@ -132,13 +132,13 @@ TraceExporter = ""
 # TraceExporterURL is the URL to which Athens populates distributed tracing
 # information such as Jaeger. In Stackdriver, use this as the GCP ProjectID.
 # Env override: ATHENS_TRACE_EXPORTER_URL
-TraceExporterURL = "http://localhost:14268"
+TraceExporterURL = ""
```

Comment:
Why did we remove this? It's a sane default since our trace backend is Jaeger by default. However, Jaeger is off but once it's on we don't need to specify the URL